composer.json

```diff
@@ -24,21 +24,21 @@
         "egulias/email-validator": "^2.1.10",
         "erusev/parsedown": "^1.7",
         "league/flysystem": "^1.0.8",
-        "monolog/monolog": "^2.0",
+        "monolog/monolog": "^2.17",
```

Comment:
@GrahamCampbell Monolog 2.17 doesn't even exist. This is a typo, this change should've been made for Carbon, not Monolog :)

Reply:
Yes.